lib/ansible/modules/crypto/luks_device.py

```diff
@@ -115,6 +115,16 @@
             - "Will only be used when I(device) and I(label) are not specified."
         type: str
         version_added: "2.10"
+    type:
+        description:
+            - "This option allow the user explicit define the format of LUKS
+              container that wants to work with. Options are 'luks1' or 'luks2'"
```

Comment:
```suggestion
              container that wants to work with. Options are C(luks1) or C(luks2)"
```

Reply:
I always forget to use C, I, etc.. lol, sorry